app.js

```diff
@@ -0,0 +1,15 @@
+const express = require('express')
+const {getAllTopics} = require('./controllers/app.controller')
+const app = express()
+
+app.use(express.json())
```

Comment:
we dont need this yet as we havent had to deal with parsing any JSON data sent to our server